tic-tac-toe/js/main.js

```diff
@@ -1,95 +1,96 @@
-var area = document.getElementById('area');
-var cell = document.getElementsByClassName('cell');
-var currentPlayer = document.getElementById('curPlyr');
+var area = document.getElementById("area");
+var cell = document.getElementsByClassName("cell");
+var currentPlayer = document.getElementById("curPlyr");
 
 var player = "x";
 var stat = {
-    'x': 0,
-    'o': 0,
-    'd': 0
-}
+  x: 0,
+  o: 0,
+  d: 0,
+};
 var winIndex = [
-    [1,2,3],
-    [4,5,6],
-    [7,8,9],
-    [1,4,7],
-    [2,5,8],
-    [3,6,9],
-    [1,5,9],
-    [3,5,7]
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9],
+  [1, 4, 7],
+  [2, 5, 8],
+  [3, 6, 9],
+  [1, 5, 9],
+  [3, 5, 7],
 ];
-
-for(var i = 1; i <= 9; i++) {
-    area.innerHTML += "<div class='cell' pos=" + i + "></div>";
+// don`t put html to all html page. You can modify html with js instead of format html in js
+for (var i = 1; i <= 9; i++) {
+  area.innerHTML += "<div class='cell' pos=" + i + "></div>";
 }
 
-for (var i = 0; i< cell.length; i++) {
-    cell[i].addEventListener('click', cellClick, false);
+for (var i = 0; i < cell.length; i++) {
+  cell[i].addEventListener("click", cellClick, false);
 }
 
 function cellClick() {
+  // don`t use var
+  var data = [];
 
-    var data = [];
-    
-    if(!this.innerHTML) {
-        this.innerHTML = player;
-    }else {
-        alert("Клітинка вже зайнята(");
-        return;
-    }
+  if (!this.innerHTML) {
+    this.innerHTML = player;
+  } else {
+    alert("Клітинка вже зайнята(");
+    return;
+  }
 
-    for(var i in cell){
-        if(cell[i].innerHTML == player){
-            data.push(parseInt(cell[i].getAttribute('pos')));
-        }
+  for (var i in cell) {
+    if (cell[i].innerHTML == player) {
+      data.push(parseInt(cell[i].getAttribute("pos")));
     }
+  }
 
-    if(checkWin(data)) {
-        stat[player] += 1;
-        restart("Виграв: " + player);
-    }else {
-        var draw = true;
-        for(var i in cell) {
-            if(cell[i].innerHTML == '') draw = false;
-        }
-        if(draw) {
-            stat.d += 1;
-            restart("Нічия");
-        }
+  if (checkWin(data)) {
+    stat[player] += 1;
+    restart("Виграв: " + player);
+  } else {
+    var draw = true;
+    for (var i in cell) {
+      if (cell[i].innerHTML == "") draw = false;
+    }
+    if (draw) {
+      stat.d += 1;
+      restart("Нічия");
     }
+  }
 
-    player = player == "x" ? "o" : "x";
-    currentPlayer.innerHTML = player.toUpperCase();
+  player = player == "x" ? "o" : "x";
+  currentPlayer.innerHTML = player.toUpperCase();
 }
 
 function checkWin(data) {
-    for(var i in winIndex) {
-        var win = true;
-        for(var j in winIndex[i]) {
-            var id = winIndex[i][j];
-            var ind = data.indexOf(id);
+  // don`t use var
```

Comment:
// don`t use var
  // in this case variable with vat can be called globally. That`s why use const or let